app/queries/items_by_storage_collection_and_quantity.rb

```diff
@@ -0,0 +1,29 @@
+module Query
+  class ItemsByStorageCollectionAndQuantity
+    attr_reader :organization, :filter_params, :items_by_storage_collection_and_quantity
+
+    def initialize(organization:, filter_params:)
+      @organization = organization
+      @filter_params = filter_params
+    end
+
+    def call
+      @items_by_storage_collection ||= ItemsByStorageCollection.new(organization: organization, filter_params: filter_params).call
```

Comment:
Why run the query again? trying to keep the logic as simple as possible - "run this other query object, and massage its results".

Why make a second query object? Trying to set the precedent that these things should be as small as possible, and it's fine for them to reference each other. I'm slightly terrified that if I make this into a single query object, next thing we know all kinds of random queries will get stapled into a single query object.